Felienne/hedy
# [UI IMPROVEMENT] Improves admin UI, add filtering and attribute selection

**Description**
In this PR we give the admin page an overall UI update to better align with the rest of the Hedy website styling. We also add dynamical attribute selection and query filtering on `creation` and `last login` date. The admin can now select either of these options and a corresponding start and end date to filter the users. The "empty" query automatically returns the last 500 users, the filtering function is without a limit. Currently we only support filtering on these two date value, but more can be added in the future.

**Fix for**
This PR fixes #1734.

**How to test**
Make sure you are an admin by setting the `os.getenv('ADMIN_USER')` value to your own username. Log in and navigate to `/admin`. Notice the new UI and verify that all the dynamic attribute selection and filtering on date is working.
